extensions/amp-audio/amp-audio.md

```diff
@@ -65,6 +65,10 @@ For example:
 
 Required if no `<source>` children are present. Must be HTTPS.
 
+##### preload
+
+If present, specifies if and how the author thinks that the audio file should be loaded when the page loads.
```

Comment:
I would add something to the effect of "same as `preload` attribute in the html `<audio>` tag (similar to what we do with [amp-img](https://github.com/ampproject/amphtml/blob/00dd0c5413a82430e55a49c1ab516c8eaaec101c/builtins/amp-img.md) ) to distinguish from amp-specific attributes.

Reply:
Change made in latest commit.